_includes/analytics.html

```diff
@@ -1,5 +1,9 @@
-	<!-- Google Analytics -->
-
+<!-- Google Analytics -->
+	<!-- GA4 -->
+	<!-- Google tag (gtag.js) --> <script async src="https://www.googletagmanager.com/gtag/js?id=G-HBYXWFP794"></script> 
+	<script> window.dataLayer = window.dataLayer || []; function gtag(){dataLayer.push(arguments);} gtag('js', new Date()); gtag('config', 'G-HBYXWFP794'); </script>
+	
+	<!-- UA -->
 	<script>
 		window.ga=window.ga||function(){(ga.q=ga.q||[]).push(arguments)};ga.l=+new Date;
 		ga('create', '{{ site.google_analytics_ua }}', 'gsa.gov');
```

Comment:
No issues with keeping the old code, but some questions.

1. Will this conflict with [GA4 guideance](https://support.google.com/analytics/answer/9744165?hl=en#zippy=%2Cin-this-article:~:text=You%20will%20see%20some%20of%20the%20following%20options%2C%20depending%20on%20how%20the%20site%20is%20configured.)?
2. UA is deprecated, should we remove lines `6-9`?
3. How can we ensure the new code is working?

Reply:
1. Based on the guidance, the new GA4 property works to collect data alongside the existing UA property, which remains unchanged. Original instructions from Dan in #2390 mentioned keeping the UA tag. That said...
2. UA does appear to no longer be collecting new data as of July 1, 2023. I haven't seen a definitive, "Yes, this is no longer needed" but as long as we don't need this tag here to _access_ the API or it's data, then we should be good to remove.
3. There are two things we'll need to check to make sure the code is working
  a. The scrips should be loading and recognized in the network tab _(See testing instructions)_
  b. We'll need Dan to log into https://analytics.google.com/ and either finish any additional setup or confirm that the data is received.

Reply:
Re: 2., I logged in and as far as I can see, new data is still being collected, FWIW.